src/xdocs/google_style.xml

```diff
@@ -78,8 +78,9 @@
               <tr>
                 <th/>
                 <th>Google's Java Style Rule</th>
-                <th>Checkstyle Check</th>
-                <th>Applied to config</th>
+                <th>Checkstyle Checks</th>
+                <th>Check configs</th>
+                <th>Sample files</th>
```

Comment:
IMO, `Checkstyle Checks` => `Checkstyle Checks Used`.

I would remove `Check configs` and add this to everything under `Checkstyle Checks Used`. Example:

````
* {Check Name}
    config
* {Check Name}
    config
````

I just don't like seeing the same names duplicated twice in both columns. It almost makes the column pointless unless there were differences between the 2.

Reply:
Or `{Check Name} (config)` .

Reply:
@Zopsss , please use this ∆∆ suggestion

Reply:
new UI:

https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/01a6e9e_2024060500/google_style.html#Google_Coverage_table

@rnveach can you confirm the coverage table look?

I have also added a new section about explaining sample files, mentioned here: https://github.com/checkstyle/checkstyle/pull/15613/files#r1744858941

Reply:
I like this.

Reply:
The table looks better.

> ATTENTION: Links to config and test in following table reference to latest 

There are no test links.

Reply:
There is no value in links to tests, nobody needs them.

We keep Links to Input as sample data.

I don't see where quoted text comes from

Reply:
> There are no test links.

removed word `test` from it:

> ATTENTION: Links to config in following table reference to latest (not released yet) config.

Reply:
> I don't see where quoted text comes from

It was from `ATTENTION` section, at the starting of Coverage Table section